tests/generic_config_updater/test_monitor_config.py

```diff
@@ -0,0 +1,135 @@
+import logging
+import pytest
+
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_res_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+logger = logging.getLogger(__name__)
+
+MONITOR_CONFIG_ACL_TABLE = "EVERFLOW_DSCP_TEST"
+MONITOR_CONFIG_ACL_RULE = "RULE_1"
+MONITOR_CONFIG_MIRROR_SESSION = "mirror_session_dscp_test"
+
+@pytest.fixture(scope='module')
+def get_valid_acl_ports(cfg_facts):
+    """ Get valid acl ports
```

Comment:
The valid port is those ports that could be associated with new added ACL_TABLE.
It is actually simulating the action of `sudo config acl add table EVERFLOW_DSCP MIRROR_DSCP`, when you don't specify the ports associated, it will add all valid ports to the ACL_TABLE `EVERFLOW_DSCP `.

Reply:
Then what ports could be associated with new added ACL_TABLE?
Please add it as code comments.